.travis.yml

```diff
@@ -1,11 +1,13 @@
 language: python
 python:
     - "2.7"
-install:
-    - pip install enum34 dateutils requests junit_xml jenkinsapi beaker-client
-                  pycodestyle
 script:
-    # Make sure all Python files are loadable
-    - find -name '*.py' | xargs -l1 -IFILE python -c 'import imp; imp.load_source("", "FILE")'
-    # Check coding style
-    - pycodestyle .
+    # Install sktm using pip to ensure dependencies are downloaded correctly.
+    - pip install .
```

Comment:
Could you put this into the `install` section, so that testing doesn't proceed if we can't have our dependencies installed?